pkg/gofr/datasource/cassandra/mock_metrics.go

```diff
@@ -6,7 +9,6 @@
 //	mockgen -source=metrics.go -destination=mock_metrics.go -package=cassandra
 //
 
-// Package cassandra is a generated GoMock package.
```

Comment:
Same doubt here ...

Reply:
Yes, I believe it would be necessary as any added comment above the import would be reflected in the godocs.
So imo it would be better to remove them from all mock related files namely (metrics,logger,interface).
For reference I have also attached screenshots of how changes are being reflected in my local machine.
<img width="675" alt="Screenshot 2024-10-14 at 1 39 46 PM" src="https://github.com/user-attachments/assets/009e091b-5192-4a87-a420-53019e658892">
<img width="675" alt="Screenshot 2024-10-14 at 1 41 51 PM" src="https://github.com/user-attachments/assets/b0067cea-38d7-4cd6-9b59-69ffbf255255">